xdsl/dialects/smt.py

```diff
@@ -218,6 +229,73 @@ class EqOp(VariadicPredicateOp):
     name = "smt.eq"
 
 
+class QuantifierOp(IRDLOperation, ABC):
+    result = result_def(BoolType())
+    body = region_def("single_block")
+
+    traits = traits_def(Pure())
+
+    assembly_format = "attr-dict-with-keyword $body"
+
+    def __init__(self) -> None:
```

Comment:
Would we not usually allow a Region/block to be put in the initialiser?

Reply:
Yeah good point, I changed it to require the region